core/src/toga/style/pack.py

```diff
@@ -53,15 +57,23 @@
 # Declaration choices
 ######################################################################
 
+# Define here, since they're not available in Travertino 0.3.0
+START = "start"
+END = "end"
+# Used in backwards compatibility section below
+ALIGNMENT = "alignment"
+ALIGN_ITEMS = "align_items"
+
 DISPLAY_CHOICES = Choices(PACK, NONE)
 VISIBILITY_CHOICES = Choices(VISIBLE, HIDDEN)
 DIRECTION_CHOICES = Choices(ROW, COLUMN)
-ALIGNMENT_CHOICES = Choices(LEFT, RIGHT, TOP, BOTTOM, CENTER)
+ALIGN_ITEMS_CHOICES = Choices(START, CENTER, END)
+ALIGNMENT_CHOICES = Choices(TOP, RIGHT, BOTTOM, LEFT, CENTER)
```

Comment:
Would be good to flag here that this is part of a deprecated declaration.

Reply:
Will do. I hadn't noticed till just now that I changed the order of directions when adding this back.